src/resources/hosts/ornl/customlauncher

```diff
@@ -250,6 +250,16 @@ class ORNLLauncher(MainLauncher):
         self.jaguar = -1
         self.rhea = -1
         self.summit = -1
+        self.andes = -1
+
+    def IsRunningOnAndes(self):
+        if self.andes == -1:
+           self.andes = 0
+           if self.parallelArgs.parallel and \
+              self.sectorname().startswith("andes-") and \
+              self.domainname() == "olcf.ornl.gov":
+              self.andes = 1
+        return self.andes
 
     def IsRunningOnRhea(self):
```

Comment:
Did you want to remove "IsRunningOnRhea"? This would include removing the call to "isRunningOnRhea" and associated code along with references to "self.rhea". I also see code related to lens and jaguar. I don't know if those still exist as well.

Reply:
That's likely a good idea, there is a fair amount of crusty code in there.